docs/reference/api/connectivity/networksocket/networksocket.md

```diff
@@ -43,15 +43,63 @@ sock.recv(buf, 100);
 sock.close();
 ```
 
-### Network socket classes
+### Changes in Mbed OS 5.10
```

Comment:
We should probably have Product review the wording of this section. @iriark01 Do you agree?

Reply:
Probably, yes. It seems to be a pretty big change in its own way.

Reply:
@maansaari Could you please review this section to make sure we're positioning this change properly from a Product perspective?

Reply:
These changes are along direction we want to go with the Socket interface, especially with secure sockets and "non-IP" sockets, so this will make the Socket higher level abstraction, which servers to future needs.